data/human/news.txt

```diff
@@ -406,7 +406,7 @@ news "unemployed youth"
 			"Actually, I'm working on a novel. Well, right now I'm just doing world-building and research, so I haven't written much of the actual story yet."
 			"Do you know how lucky you are, that you can visit a different world every week if you want to? I've been stuck on this one filthy little planet my whole life."
 			"If I had enough money, I'd go up to a captain in the spaceport and ask for transport to anywhere else. Nowhere's as bad as this place."
-			"Are you hiring new crewmembers? And are you by chance planning to fly to the Paradise Worlds?"
+			"Are you hiring new crew members? And are you by chance planning to fly to the Paradise Worlds?"
```

Comment:
```suggestion
			"Are you hiring new crewmembers? And are you by chance planning to fly to the Paradise Worlds?"
```

"[crewmember](https://en.wiktionary.org/wiki/crewmember)" is actually fine as a gender neutral form of "crewman".